[2025-03-30T10:39:37.284+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T10:39:37.383+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aqi_etl_pipeline.get_aqi_data_Bangkok manual__2025-03-30T10:39:26.348292+00:00 [queued]>
[2025-03-30T10:39:37.466+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aqi_etl_pipeline.get_aqi_data_Bangkok manual__2025-03-30T10:39:26.348292+00:00 [queued]>
[2025-03-30T10:39:37.466+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-03-30T10:39:37.593+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_aqi_data_Bangkok> on 2025-03-30 10:39:26.348292+00:00
[2025-03-30T10:39:37.703+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=52980) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-30T10:39:37.713+0000] {standard_task_runner.py:72} INFO - Started process 53062 to run task
[2025-03-30T10:39:37.707+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'aqi_etl_pipeline', 'get_aqi_data_Bangkok', 'manual__2025-03-30T10:39:26.348292+00:00', '--job-id', '505', '--raw', '--subdir', 'DAGS_FOLDER/api_etl_pipeline.py', '--cfg-path', '/tmp/tmpdxkz4zy9']
[2025-03-30T10:39:37.723+0000] {standard_task_runner.py:105} INFO - Job 505: Subtask get_aqi_data_Bangkok
[2025-03-30T10:39:37.953+0000] {task_command.py:467} INFO - Running <TaskInstance: aqi_etl_pipeline.get_aqi_data_Bangkok manual__2025-03-30T10:39:26.348292+00:00 [running]> on host 58b9640d9ecf
[2025-03-30T10:39:38.404+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aqi_etl_pipeline' AIRFLOW_CTX_TASK_ID='get_aqi_data_Bangkok' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T10:39:26.348292+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T10:39:26.348292+00:00'
[2025-03-30T10:39:38.409+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T10:39:38.409+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T10:39:38.410+0000] {logging_mixin.py:190} INFO - Current task name:get_aqi_data_Bangkok state:running start_date:2025-03-30 10:39:37.392926+00:00
[2025-03-30T10:39:38.410+0000] {logging_mixin.py:190} INFO - Dag name:aqi_etl_pipeline and current dag run status:running
[2025-03-30T10:39:38.410+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-30T10:40:12.122+0000] {logging_mixin.py:190} INFO - ‚ùå API request failed for city Bangkok. Status code: 429
[2025-03-30T10:40:12.124+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T10:40:12.132+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-30T10:40:12.132+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=aqi_etl_pipeline, task_id=get_aqi_data_Bangkok, run_id=manual__2025-03-30T10:39:26.348292+00:00, execution_date=20250330T103926, start_date=20250330T103937, end_date=20250330T104012
[2025-03-30T10:40:12.139+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T10:40:12.139+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T10:40:12.139+0000] {logging_mixin.py:190} INFO - Dag name:aqi_etl_pipeline queued_at:2025-03-30 10:39:26.372196+00:00
[2025-03-30T10:40:12.139+0000] {logging_mixin.py:190} INFO - Task hostname:58b9640d9ecf operator:PythonOperator
[2025-03-30T10:40:12.176+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T10:40:12.209+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T10:40:12.211+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
